src/net/sourceforge/kolmafia/request/DeckOfEveryCardRequest.java

```diff
@@ -199,8 +199,18 @@ public static EveryCard buffToCard(AdventureResult buff) {
     return DeckOfEveryCardRequest.buffToCard.get(buff);
   }
 
+  // Only used in testing.  Note can return null.
+  public static EveryCard getCardById(int id) {
```

Comment:
anything used only by tests doesn't have to be public: it can be package-private (i.e. no modifier)

This keeps the public interface for the class minimal.

Reply:
Done.  I got in the habit of public because once something didn't work until I made it public.  I suspect I needed something from another package in a test but I established the wrong habit :-)